File: SGE/ListadoPersonasYDepartamentos/Domain/Interfaces/IUseCaseJuego.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Interfaces
{
    using Domain.DTOs;
    using Domain.Entities;
    using System.Collections.Generic;

    public interface IUseCaseJuego
    {
        DatosJuegoDTO ObtenerDatosJuego();
        ResultadoJuego ComprobarDepartamentos(Dictionary<int, int> asignaciones);
    }
}

File: DI/tema8/Select-persona-Azure/app/domain/interfaces/use-cases/IUseCaseJuego.ts
import { DatosJuegoDTO } from '../../dtos/DatosJuegoDTO';
import { ResultadoJuego } from '../../entities/ResultadoJuego';

export interface IUseCaseJuego {
  obtenerDatosJuego(): Promise<DatosJuegoDTO>;
  comprobarDepartamentos(asignaciones: { [key: number]: number }): Promise<ResultadoJuego>;
}
